packs/cloudcasa-3.1.0/values.yaml

```diff
@@ -0,0 +1,18 @@
+pack:
+  # The namespace (on the target cluster) in which the pack will be deployed.
+  # When the namespace is not found, a new namespace will be created with the name specified.
+  # Note this namespace MUST NOT BE CHANGED from "cloudcasa-io".
```

Comment:
Include the content.images in the pack.  Check other packs for examples.  This should be the image or images needed for the pack to functions.  This is how we use packs in airgap to bundle the images listed here.
https://github.com/spectrocloud/pack-central/blob/main/packs/kyverno-1.12.2/values.yaml